Wikipedia/assets/codemirror/codemirror-black-none.css

```diff
@@ -0,0 +1,95 @@
+html, body, .CodeMirror {
```

Comment:
@tonisevener it should be possible to have a single `codemirror-syntax-highlighting-off.css` that has all of the css classes with non-default text colors set to `color: inherit`. This separate css could be linked below the `codemirror-theme` css and toggled on and off with the href switching between `"codemirror-syntax-highlighting-off.css"` and `""`

Reply:
Also, just to clarify, this isn't a required suggestion since the approach here works and affords more flexibility, just mentioning in case it's helpful

Reply:
@joewalsh ah no I like that, inherit was what I was missing. I wasn't sure how to turn off syntax coloring but still keep the base theme color without quadrupling up on css files. I'll change, thanks!